app/controllers/concerns/notification_endpoint.rb

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
This module might benefit from top-level documentation explaining its main purpose.